components/codeflare/codeflare.go

```diff
@@ -71,9 +72,32 @@ func (c *CodeFlare) GetComponentName() string {
 	return ComponentName
 }
 
+func (c *CodeFlare) UpdateStatus(in *status.ComponentsStatus) error {
+	codeFlareStatus, err := c.GetReleaseVersion(in, deploy.DefaultManifestPath, ComponentName)
+
+	if err != nil {
+		in.CodeFlare = &status.CodeFlareStatus{}
+		return err
+	}
+
+	in.CodeFlare = &status.CodeFlareStatus{
+		ComponentStatus: status.ComponentStatus{
+			UpstreamReleases: []status.ComponentReleaseStatus{{
+				Name:        status.Platform(ComponentName),
+				DisplayName: CodeflareOperator,
```

Comment:
any reason to keep both?
or maybe share the vision after this 1st phase, what will be built evenetually so we do not need to deprcated fields later once we add here

Reply:
@zdtsw this was discussed in the refinement [here](https://docs.google.com/document/d/1nbQB-uA48x79Ci3xrpMHGfdo3XjTR4xirtWk0we0kl8/edit?disco=AAABP_8ne1g)